README.rst

```diff
@@ -38,6 +38,7 @@ If you want to get involved head over to GitHub to get the source code, look at
 * `GitHub page`_
 * `Development documentation`_
 * `Development IRC`_
+* ``#pip`` on the `PyPA Discord`_
```

Comment:
I'd prefer to leave this out of the README, TBH.

Reply:
> I'd prefer to leave this out of the README, TBH.

removed: [2b17408](https://github.com/pypa/pip/pull/12840/commits/2b174087d25096538112f15cce8fd949fac816ff)